cdap-docs/admin-manual/source/cdap-hadoop-compatibility.rst

```diff
@@ -24,63 +24,65 @@ compatibility or know to be incompatible.
-| 3.3.0          | **No**  | **No**  | **No**  | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** |
-+----------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
-| 3.2.x          | **No**  | **No**  | **No**  | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | **Yes** | **Yes** | **Yes** | **Yes** |
-+----------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
-| 3.1.x          | **No**  | **No**  | **No**  | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | **Yes** | **Yes** | **Yes** | **Yes** |
-+----------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
-| 3.0.x          | **No**  | **No**  | **No**  | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | **Yes** | **Yes** | **Yes** |
-+----------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
-| 2.8.x          | **No**  | **No**  | **No**  | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    | *No*    |
-+----------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
++----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| **CDH Distro Versions**                                                                                                                                              |
++================+=========+=========+=========+=========+=========+=========+=========+=========+=========+=========+=========+=========+=========+=========+=========+
+| CDAP Version   | 5.15    | 5.14    | 5.13    | 5.12    | 5.11    | 5.10    | 5.9     | 5.8     | 5.7     | 5.6     | 5.5     | 5.4     | 5.3     | 5.2     | 5.1     |
++----------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
+| 6.0.0          | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **No**  | **No**  | **No**  |
++----------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
+| 5.1.0          | **No**  | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **Yes** | **No**  | **No**  | **No**  |
```

Comment:
technically we have been including 5.15 in our integration tests since Jun 28, 2018.  So we likely support it in 5.1.x, but i'm ok leaving it out unless we can confirm

Reply:
I dont see 5.15 in release/5.1 ITN https://builds.cask.co/browse/IT-ITN7-34. So will keep as is for now.